ml_service/pipelines/diabetes_regression_build_train_pipeline_with_r.py

```diff
@@ -26,15 +26,11 @@ def main():
         print("aml_compute:")
         print(aml_compute)
 
-    run_config = RunConfiguration(conda_dependencies=CondaDependencies.create(
-        conda_packages=['numpy', 'pandas',
-                        'scikit-learn', 'tensorflow', 'keras'],
-        pip_packages=['azure', 'azureml-core',
-                      'azure-storage',
-                      'azure-storage-blob'])
-    )
+    # Create a run configuration environment
+    conda_deps_file = "diabetes_regression/training_dependencies.yml"
+    conda_deps = CondaDependencies(conda_deps_file)
+    run_config = RunConfiguration(conda_dependencies=conda_deps)
     run_config.environment.docker.enabled = True
```

Comment:
we will need that container because it has some r_essential installed on it. @eedorenko please confirm?

Reply:
Confirm

Reply:
Indeed, just running as-is yields
```
Starting the daemon thread to refresh tokens in background for process with pid = 137
Entering Run History Context Manager.
sh: 1: Rscript: not found
```

I will test when changing the training conda yml

Reply:
Fixed in other PR